operators/transformation/reduce.md

```diff
@@ -0,0 +1,26 @@
+# reduce
+#### signature: `reduce(accumulator: function, seed: any): Observable`
+
+## Reduces the values from source observable to a single value that's emitted when the source completes.
+
+> Just like `Array.prototype.reduce()`
+
+### Examples
+
+##### Example 1: Sum a stream of numbers
+
+( [jsBin](http://jsbin.com/dakuneneho/edit?js,console) | [jsFiddle](https://jsfiddle.net/f8fw7yka/) )
+
+```js
+const source = Rx.Observable.of(1, 2, 3, 4);
+const example = source.reduce((acc,val) => acc + val);
+//output: Sum: 10'
+const subscribe = example.subscribe(val => console.log('Sum:', val));
+```
+
+### Additional Resources
+* [concatMap](http://reactivex.io/rxjs/class/es6/Observable.js~Observable.html#instance-method-reduce) :newspaper: - Official docs
```

Comment:
This looks great, just one update. Can you change the link text here from `concatMap` to `reduce`? Thanks!